usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -77,6 +77,12 @@ if test "$BACKUP" = "TSM" ; then
     # see https://github.com/rear/rear/issues/1533
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$TSM_LD_LIBRARY_PATH
 fi
+if test "$BACKUP" = "FDRUPSTREAM" ; then
```

Comment:
I think this part about BACKUP="FDRUPSTREAM"
does not belong to "support for s390 (zLinux) arch"
so that it should be removed from this pull request.

Reply:
correct, this will be pulledout